db/migrations/20191202153352_initial.js

```diff
@@ -1,25 +1,32 @@
 
 exports.up = function(knex) {
   return Promise.all([
-    knex.schema.createTable('projects', table => {
+    knex.schema.createTable('users', table => {
       table.increments('id').primary();
-      table.string('project_name');
+      table.text('user_name');
+      table.text('password');
+
+      table.timestamps(true, true);
+    }),
+
+    knex.schema.createTable('projects', table => {
+      table.increments('id').primary();      
+      table.text('project_name');
       table.unique('project_name');
-      table.integer('palette0').unsigned();
-      table.integer('palette1').unsigned();
-      table.integer('palette2').unsigned();
+      table.integer('user_id').references('users.id')
```

Comment:
Should this be "table.integer('user_id')" on this line and then "table.foreign('user_id').references('users.id')?